kong/plugins/zipkin/tracing_headers.lua

```diff
@@ -314,6 +315,35 @@ local function parse_jaeger_trace_context_headers(jaeger_header)
 end
 
 
+local function parse_datadog_headers(headers)
+  local warn = kong.log.warn
+
+  local should_sample = headers["x-datadog-sampling-priority"]
+  if should_sample == "1" or should_sample == "2" then 
+    should_sample = true
+  elseif should_sample == "0" or should_sample == "-1" then
+    should_sample = false
+  elseif should_sample ~= nil then
+    warn("x-datadog-sampling-priority header invalid; ignoring.")
+    return nil, nil, nil
+  end
+
+  local trace_id = headers["x-datadog-trace-id"]
```

Comment:
nit: Other headers are named: "ot-tracer-traceid""x-b3-traceid"
Shall we keep naming convention here?

Reply:
The header name is defined by DataDog, we should keep this.

Reply:
sounds good.